packages/ERTP/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.11.12](https://github.com/Agoric/agoric-sdk/compare/@agoric/ertp@0.11.10...@agoric/ertp@0.11.12) (2021-08-14)
+
+### 0.26.10 (2021-07-28)
+
+
+### Bug Fixes
+
+* **ERTP:** use metered=true and xs-worker on all swingset tests ([8c3da1f](https://github.com/Agoric/agoric-sdk/commit/8c3da1fa05c5734e1c839d480642f1716d003dd3))
```

Comment:
that seems more like a `test` than a `fix`; i.e. it doesn't seem visible to API clients.

Is ERTP a CHANGELOG package or a NEWS package? Ah NEWS. OK.